File: src/main/java/com/mercateo/jsonschema/mapper/ObjectNodeFactory.java
package com.mercateo.jsonschema.mapper;

import com.fasterxml.jackson.databind.node.JsonNodeFactory;
import com.fasterxml.jackson.databind.node.ObjectNode;

public class ObjectNodeFactory {

    public static final JsonNodeFactory nodeFactory = new JsonNodeFactory(true);

    public ObjectNode createNode() {
        return new ObjectNode(nodeFactory);
    }
}

File: src/main/java/com/mercateo/jsonschema/mapper/ObjectNodeFactory.kt
package com.mercateo.jsonschema.mapper

import com.fasterxml.jackson.databind.node.JsonNodeFactory
import com.fasterxml.jackson.databind.node.ObjectNode

class ObjectNodeFactory {

    fun createNode(): ObjectNode {
        return ObjectNode(nodeFactory)
    }

    companion object {
        val nodeFactory = JsonNodeFactory(true)
    }
}
